### DISCUSSION TITLE
I fit the esp32 into the DualShock 2 plug. The size has unchanged.

### DISCUSSION THREAD

**Discussion by AntonLesnichin** [darthcloud/BlueRetro#599] [Show and tell]
Hi, my name is Yuki!

I fit the esp32 d1 mini into the regular DualShock 2 plug. It's a really neat design project because I didn't resize a plug and didn't make extra holess. I do not use 3d technologies, only manual work for the soul. And my device looks great! 

I recorded the whole process of making and also show an animation schematic of how to solder it youself.

Enjoy watching!
https://youtu.be/u01V9mxeGcE

P.s. English subtitles are now available!

![PXL_20230117_175123899](https://user-images.githubusercontent.com/36990248/216995875-59a4b471-14b9-4af1-af5f-1ef54c680937.jpg)
![PXL_20230117_175839217 (1)](https://user-images.githubusercontent.com/36990248/216995866-5693cce9-79bf-4b27-8fc7-3452907d9f17.jpg)
![PXL_20230117_173733827 (1)](https://user-images.githubusercontent.com/36990248/216995871-5d13f122-f3a0-4c62-a5e3-d101c118f103.jpg)
![PXL_20230117_175544623](https://user-images.githubusercontent.com/36990248/216995861-0eeac2f3-05b6-4c79-9354-8ee31010ed74.jpg)

**Comment by zeus86** [comment]
Just watched the video with autogenerated subtitles. This is really georgeous, but i just wonder, why you just use a bare esp-module? they are smaller, and you can preflash them with a cheap esp-flasher, and don't need all of that dev-board-stuff like usb-ports. a 5v-to-3v3-ams1117 and the leds are probably all you need and you don't need to butcher the board ^^

**Comment by darthcloud** [comment]
Nice work thanks for sharing!
